core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -183,6 +204,46 @@ var ExplorationEditorTranslationTab = function() {
     expect(numericalStatus.getText()).toMatch(content);
   };
 
+  this.expectNumericalStatusAccessibilityToMatch = function(content) {
+    expect(numericalStatus.getAttribute(
+      'aria-label')).toMatch(content);
```

Comment:
This might fit on one line I think.

Reply:
I have made it like that inorder make the code look clean. Should I change it @nithusha21

Reply:
If it fits on a single line, please make it a single line (here and everywhere else).